public/mockServiceWorker.js

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable */
+ 
```

Comment:
```suggestion
/* eslint-disable */
```

This has been added to "ignore" in eslint config file. Don't know why there is an error thrown on CI, when locally linting is successful.